### ISSUE TITLE
[links] insufficient clickable area of navigation links

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#170]
URL: https://openreferraluk.org

Include some padding surrounding the navigation links to make it easier for users that are unable to maintain stable mouse movements. Some margin top and bottom will need to be removed to compensate.

**Comment by UsmanAfzal** [comment]
Checked and approved
